src/Test/EntityManagerMockFactory.php

```diff
@@ -20,12 +20,21 @@
 use Doctrine\ORM\EntityRepository;
 use Doctrine\ORM\QueryBuilder;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\EntityManagerMockFactory class is deprecated since 1.x in favor of '.
+    __NAMESPACE__.'\EntityManagerMockFactory, and will be removed in 2.0.',
```

Comment:
:thinking: it's… the same class twice?!
```suggestion
    __NAMESPACE__.'\EntityManagerMockFactoryTrait, and will be removed in 2.0.',
```

Reply:
Fixed